src/IterTools.jl

```diff
@@ -595,7 +595,7 @@ subsets(xs::C, ::Val{K}) where {K, C} = StaticSizeBinomial{K, C}(xs)
 iterate(it::StaticSizeBinomial{0}, state=false) = state ? nothing : ((), true)
 
 # Generic case K >= 1
-pop(t::NTuple) = reverse(tail(reverse(t))), t[end]
+pop(t::NTuple) = Base.front(t), last(t)
```

Comment:
why not
```suggestion
pop(t::NTuple) = first(t), last(t)
```
Which avoids using the internal `Base.front`  method?

Reply:
Docs link: https://docs.julialang.org/en/v1/base/collections/#Base.front

Reply:
fair enough.
And it exists in 1.6